CHANGES.txt

```diff
@@ -1,3 +1,8 @@
+V3.5.1
```

Comment:
Could you please adapt this?

Next release will be 3.6.0 and has already a list of changes.